students/20th/garamsong/login/login.css

```
@@ -0,0 +1,73 @@
+@import url('https://fonts.googleapis.com/css2?family=Lobster&display=swap');
+
+
+body{
+    display:flex;
+    justify-content:center;
```

```suggestion
body {
    display: flex;
    justify-content: center;
```
띄어쓰기를 적절하게 사용해주시면 가독성에 도움이 될것 같습니다!

---

확인 후 수정 완료 했습니다